src/commands/give-up/index.ts

```diff
@@ -17,7 +17,12 @@ const giveUp = async (
   context: Context,
   challengeIssueService: ChallengeIssueService
 ) => {
-  const issueResponse = await context.github.issues.get(context.issue());
+  const issue = context.issue();
+  const issueResponse = await context.github.issues.get({
+    owner: issue.owner,
```

Comment:
I think it can be `...context.issue()`.

Reply:
Good suggest

Reply:
I try to write in your way, but when I use the `...` operator, it will pass the `number` to the parameter, that will trigger the deprecation tips appears. I write this mainly to disappear it.